static/docs/use-cases/huge-data-on-an-external-drive.md

```diff
@@ -0,0 +1,156 @@
+
+```dvc
+$ sudo chown <username>: -R /mnt/data/
+$ chmod u+rw -R /mnt/data/
+```
+
+## Start a DVC project and setup a local external cache
+
+An _external_ <abbr>cache</abbr> is called so because it resides outside of your
+<abbr>workspace</abbr> directory. We also call it _local_ because it is located
+within our filesystem (as opposed to being located somewhere on the internet, in
+which case it is called _remote_). Let's create a directory for it:
+
+```dvc
+$ mkdir -p /mnt/data/dvc-cache
+```
+
+Now you can initialize a project on your home directory and configure it to use
+the external cache directory:
+
+```dvc
+$ cd ~/project/
+$ git init
+$ dvc init
+
+$ dvc config cache.dir /mnt/data/dvc-cache
+$ rm -rf .dvc/cache/
+
+$ git add .dvc/config
+$ git commit -m 'DVC with external cache dir'
+```
+
+If you check the config file you will see something like this:
+
+```dvc
+$ cat .dvc/config
+[cache]
+dir = /mnt/data/dvc-cache
+```
+
+## Example of tracking external dependencies and outputs
+
+Now, when you refer to the data files and directories, you have to use their
+absolute path. The DVC-files will be created on the <abbr>project</abbr>
+directory, and you can track their modifications with `git` as usual.
+
+For example let's say that the raw data are on `/mnt/data/raw/` and you are
+cleaning them up. You could do it like this:
+
+```dvc
+$ dvc add /mnt/data/raw
+
+$ dvc run -f clean.dvc \
+          -d /mnt/data/raw \
+          -o /mnt/data/clean \
+          ./cleanup.py /mnt/data/raw /mnt/data/clean
+```
+
+<details>
+
+### Using an environment variable for the data path
+
+In a real life situation probabaly you would declare an environment variable
+`DATA_PATH=/mnt/data` and use it to shorten the command options, like this:
+
+```dvc
+$ dvc add $DATA_PATH/raw
+
+$ dvc run -f clean.dvc \
+          -d $DATA_PATH/raw \
+          -o $DATA_PATH/clean \
+          ./cleanup.py $DATA_PATH/raw $DATA_PATH/clean
+```
+
+</details>
+
+If you check the contents of the files `raw.dvc` and `clean.dvc` you will notice
+that their `path:` field refers to the external directories:
+
+```dvc
+$ cat raw.dvc
+md5: 9cbbacd47133debf91dcb41891c64730
+wdir: .
+outs:
+- md5: 0ee0a6bc0a1f1be0610f7a3f67f1cb54.dir
+  path: /mnt/data/raw
+  cache: true
+  metric: false
+  persist: false
+
+$ cat clean.dvc
+md5: 2b842ed58b1792dde6df27e3d0f73430
+cmd: cp -a /mnt/data/raw /mnt/data/clean
+wdir: .
+deps:
+- md5: 0ee0a6bc0a1f1be0610f7a3f67f1cb54.dir
+  path: /mnt/data/raw
+outs:
+- md5: 0ee0a6bc0a1f1be0610f7a3f67f1cb54.dir
+  path: /mnt/data/clean
+  cache: true
+  metric: false
+  persist: false
+```
+
+You can also check and verify that indeed all the data and cache files are
+stored on the external drive:
+
+```dvc
+$ ls /mnt/data/
+clean  dvc-cache  raw
+
+$ ls /mnt/data/dvc-cache
+. . .
+
+```
+
+Now you can add and commit the DVC-files to git:
+
+```dvc
+$ git add raw.dvc clean.dvc
+$ git commit -m "cleanup raw data"
```

Comment:
I would say that some explanation is missing that DVC will move those files to the cache we set, that it will be switching versions on `dvc checkout`. To some extent that `data` becomes kind of part of a virtual extended workspace that is managed by `dvc checkout`, `dvc pull`, etc.

Reply:
Also, there is a missing part about `cache type` importance in this case. By default we may end up copying files. Since it's an "advanced" case, users should see a reference and understand how to switch DVC to use links instead of copying.